tests/karate/src/test/java/be/utils/ScalaServer.java

```diff
@@ -23,9 +25,25 @@ public void stop() {
   }
 
   @Override
-  public String[] getCmd() {
+  public String[] getCmd() throws IOException {
     String configPath = Paths.get("src", "main", "scala", "ch", "epfl", "pop", "config").toString();
-    return new String[] { "sbt", "-Dscala.config=" + configPath, "run" };
+    File targetJar = getTargetJar();
+
+    return new String[] { "java", "-Dscala.config=" + configPath, "-jar", targetJar.getCanonicalPath()};
+  }
+
+  private File getTargetJar() throws IOException {
```

Comment:
Maybe more vertical spacing to improve readability ;-)

Reply:
Fixed

Reply:
Did my commit fail... ?